app/src/main/res/xml/notification_settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:title="@string/settings_category_notification_title">
+
+    <PreferenceCategory>
+        <SwitchPreferenceCompat
+                android:defaultValue="false"
+                android:key="@string/scale_to_square_image_in_notifications_key"
+                android:summary="@string/notification_scale_to_square_image_summary"
+                android:title="@string/notification_scale_to_square_image_title"
+                app:iconSpaceReserved="false" />
+
+    </PreferenceCategory>
```

Comment:
Maybe remove `PreferenceCategor`ies or put everything in a single one: now there is some strange spacing above and below the normal preference, so try to play around with this until the layout looks good

Reply:
The reason for the strange spacing was that i forgot to set the layout of PreferenceCategory. I just had in my mind that in the other settings the PreferenceCategory created a visual seperation, which was needed since i removed the divider.